logic/src/commonMain/kotlin/com/wire/kalium/logic/configuration/ServerConfig.kt

```diff
@@ -26,7 +27,8 @@ data class ServerConfig(
             teamsUrl = """https://teams.wire.com""",
             blackListUrl = """https://clientblacklist.wire.com/prod""",
             websiteUrl = """https://wire.com""",
-            title = "Production"
+            title = "Production",
+            androidSenderId = "782078216207"
```

Comment:
I feel like this shouldn't be the default, and that Reloaded should do something like this instead:

```kotlin
val prodConfig = PRODUCTION.copy(senderId = "782078216207")
val stagingConfig = STAGING.copy(senderId = "723990470614")

// and then call the usecases:


login(user, password, prodConfig)
```

Or even better, we should make `PRODUCTION` and `STAGING` private and expose a `fun createProduction(senderId: String)` and `fun createStaging(senderId: String)`.